zwave_js_server/model/statistics.py

```diff
@@ -79,12 +79,15 @@ def repeater_rssi(self) -> List[int]:
         return repeater_rssi
 
     @property
-    def route_failed_between(self) -> Optional[List["Node"]]:
+    def route_failed_between(self) -> Optional[Tuple["Node", ...]]:
```

Comment:
Should we specify that it's always a two item tuple?

Reply:
I tried that but with list comprehension mypy didn't like it. Should I be explicit with the indexes?

Reply:
I think we should be explicit with the typing since we know the length. The code should follow.